library/xml-model/src/test/java/org/cru/godtools/xml/model/tract/TractPageTest.kt

```diff
@@ -53,5 +53,11 @@ class TractPageTest {
         assertEquals("Modal 2", page.modals[1].title!!.text)
     }
 
+    @Test
+    fun testCardBackgroundColorFallbackBehavior() {
+        val page = parsePageXml("page_cards.xml")
+        assertEquals(page.cardBackgroundColor, manifest.cardBackgroundColor)
```

Comment:
similar to the manifest test, I want to avoid making the page cardBackgroundColor fallback behavior test dependent on parsing xml